docs/tutorials/chisel.rst

```diff
@@ -1,5 +1,5 @@
-Install chiselled slices into a ROCK
-====================================
+Put Chisel slices in a ROCK
```

Comment:
```suggestion
Install package slices in a ROCK
```

Reply:
How about `Install Chisel slices in a ROCK` then? it would fit in one line.